app/Resources/views/themes/ezplatform/full/bundle_list.html.twig

```diff
@@ -14,16 +14,12 @@
         <div class="bundles-list-search">
             <div class="container">
                 <div class="col-xs-12">
-                    {{ form_start(searchForm) }}
-                        <div class="form-group">
-                            {{ form_widget(searchForm.search, { 'id': 'bundles-list-search-query'} )}}
-                            <div class="input-group-addon">
-                                <button type="submit" class="btn bundles-list-search-button">
-                                    <i class="fa fa-search" aria-hidden="true"></i>
-                                </button>
-                            </div>
-                        </div>
-                    {{ form_end(searchForm) }}
+                    {% if search is defined %}
```

Comment:
`if` statement could be replace with single line:
`{% set searchText = search is defined ? search : '' %}`